beacon-chain/rpc/eth/rewards/handlers_test.go

```diff
@@ -525,8 +501,44 @@ func TestAttestationRewards(t *testing.T) {
 		resp := &AttestationRewardsResponse{}
 		require.NoError(t, json.Unmarshal(writer.Body.Bytes(), resp))
 		assert.Equal(t, "0", resp.Data.TotalRewards[0].Head)
-		assert.Equal(t, "-432270", resp.Data.TotalRewards[0].Source)
-		assert.Equal(t, "-802788", resp.Data.TotalRewards[0].Target)
+		assert.Equal(t, "-439299", resp.Data.TotalRewards[0].Source)
+		assert.Equal(t, "-815841", resp.Data.TotalRewards[0].Target)
```

Comment:
These 2 changed because I simplified the test to copy the initial state, and the initial state had different logic when populating the effective balance (used `i+1` instead of `i`).